src/test/csi-qingcloud/README.md

```diff
@@ -28,6 +28,11 @@ helm delete csi-qingcloud --namespace kube-system
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 
+## Upgrade Notes
+
+The qingcloud-csi chart uses snapshotter v4.0 by default starting from v1.2.9, v1 crd is required for snapshotter v4.0.0 and above, so user must install v1 crd, otherwise the snapshot operation will be stuck.
+With the update of attacher and provisioner, the minimum k8s support version has been increased to 1.17, and K8s 1.20 version is recommended.
```

Comment:
do not write like this. add a new section e.g. `support matrix`, and add a table to detail the supported k8s version and snapshot crd versions.

Reply:
Okey，thanks for your suggestion.